### ISSUE TITLE
No registered env with id: OPEn_env-v0

### ISSUE THREAD

**Issue by phunguyen1195** [chuangg/OPEn#2]
Hello,
I'm having some trouble using the environment. The error I'm having is: No registered env with id: OPEn_env-v0.

---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
~/anaconda3/envs/rlearning/lib/python3.7/site-packages/gym/envs/registration.py in spec(self, path)
    117         try:
--> 118             return self.env_specs[id]
    119         except KeyError:

KeyError: 'OPEn_env-v0'

During handling of the above exception, another exception occurred:

UnregisteredEnv                           Traceback (most recent call last)
<ipython-input-12-263240bbee7e> in <module>
----> 1 main()

<ipython-input-11-d1abc6927e4c> in main()
      4 def main():
      5     # Tdw_ip can be localhost if running on same machine else the ip address of the machine
----> 6     env = gym.make('OPEn_gym:OPEn_env-v0', asset_path="/Users/user/Downloads/3d_models/Darwin/", tdw_ip=None, self_ip="localhost", port=None, debug=False)
      7     # You have to specify config that scene objects, maximum number for them, agent position (Will be placed randomly
      8     # if not included)

~/anaconda3/envs/rlearning/lib/python3.7/site-packages/gym/envs/registration.py in make(id, **kwargs)
    140 
    141 def make(id, **kwargs):
--> 142     return registry.make(id, **kwargs)
    143 
    144 def spec(id):

~/anaconda3/envs/rlearning/lib/python3.7/site-packages/gym/envs/registration.py in make(self, path, **kwargs)
     84         else:
     85             logger.info('Making new env: %s', path)
---> 86         spec = self.spec(path)
     87         env = spec.make(**kwargs)
     88         # We used to have people override _reset/_step rather than

~/anaconda3/envs/rlearning/lib/python3.7/site-packages/gym/envs/registration.py in spec(self, path)
    126                 raise error.DeprecatedEnv('Env {} not found (valid versions include {})'.format(id, matching_envs))
    127             else:
--> 128                 raise error.UnregisteredEnv('No registered env with id: {}'.format(id))
    129 
    130     def register(self, id, **kwargs):

UnregisteredEnv: No registered env with id: OPEn_env-v0

**Comment by abhi1092** [comment]
Did you install the environment?. Can you do `pip list` and paste the output. And also did you try running the example in the documents.

**Comment by phunguyen1195** [comment]
Hello, My apologies for the late reply! 
I believe I did install the environment following the steps in README. I did run the example and the error I got is above. Below is my `pip list` . Thank you.

Package                   Version             Location                                          
------------------------- ------------------- --------------------------------------------------
absl-py                   0.9.0               
aiohttp                   3.6.2               
altgraph                  0.17.2              
appdirs                   1.4.3               
appnope                   0.1.0               
astor                     0.8.1               
async-timeout             3.0.1               
atari-py                  0.2.6               
attrs                     19.3.0              
backcall                  0.1.0               
beautifulsoup4            4.8.2               
black                     19.10b0             
bleach                    3.1.0               
boto3                     1.19.4              
botocore                  1.22.4              
cachetools                4.0.0               
certifi                   2019.11.28          
cffi                      1.13.2              
chardet                   3.0.4               
click                     7.1.1               
cloudpickle               1.3.0               
colorama                  0.4.3               
decorator                 4.4.1               
defusedxml                0.6.0               
entrypoints               0.3                 
filelock                  3.0.12              
funcsigs                  1.0.2               
future                    0.18.2              
gast                      0.2.2               
google                    2.0.3               
google-auth               1.11.2              
google-auth-oauthlib      0.4.1               
google-pasta              0.1.8               
grpcio                    1.27.2              
grpcio-tools              1.26.0              
gym                       0.17.1              
h5py                      2.10.0              
idna                      2.8                 
imageio                   2.10.1              
importlib-metadata        1.5.0               
ipykernel                 5.1.3               
ipython                   7.11.1              
ipython-genutils          0.2.0               
ipywidgets                7.5.1               
jedi                      0.15.2              
Jinja2                    2.10.3              
jmespath                  0.10.0              
jsonschema                3.2.0               
jupyter                   1.0.0               
jupyter-client            5.3.4               
jupyter-console           6.1.0               
jupyter-core              4.6.1               
Keras-Applications        1.0.8               
Keras-Preprocessing       1.1.0               
lz4                       3.0.2               
macholib                  1.15.2              
Markdown                  3.2.1               
MarkupSafe                1.1.1               
mistune                   0.8.4               
mkl-fft                   1.0.15              
mkl-random                1.1.0               
mkl-service               2.3.0               
more-itertools            8.2.0               
multidict                 4.7.5               
nbconvert                 5.6.1               
nbformat                  4.4.0               
notebook                  6.0.2               
numpy                     1.18.1              
oauthlib                  3.1.0               
open-environment-gym      0.0.1               /Users/user/Documents/GitHub/fall2021/OPEn        
opencv-python             4.2.0.32            
opencv-python-headless    4.2.0.32            
opt-einsum                3.2.0               
packaging                 20.3                
pandas                    1.0.1               
pandocfilters             1.4.2               
parso                     0.5.2               
pathspec                  0.7.0               
pexpect                   4.7.0               
pickleshare               0.7.5               
Pillow                    8.4.0               
pip                       19.3.1              
pluggy                    0.13.1              
prometheus-client         0.7.1               
prompt-toolkit            3.0.2               
protobuf                  3.11.3              
psutil                    5.8.0               
ptyprocess                0.6.0               
py                        1.8.1               
py-spy                    0.3.3               
pyasn1                    0.4.8               
pyasn1-modules            0.2.8               
pybullet                  3.1.9               
pybulletgym               0.1                 /Users/user/Documents/GitHub/fall2021/pybullet-gym
pycparser                 2.19                
pygame                    2.0.2               
pyglet                    1.5.0               
Pygments                  2.5.2               
pyinstaller               4.5.1               
pyinstaller-hooks-contrib 2021.3              
pymongo                   3.12.1              
pyparsing                 2.4.6               
pyrsistent                0.15.7              
pytest                    5.3.5               
python-dateutil           2.8.1               
pytz                      2019.3              
PyYAML                    5.2                 
pyzmq                     18.1.0              
qtconsole                 4.6.0               
ray                       0.8.2               
redis                     3.4.1               
regex                     2020.2.20           
requests                  2.22.0              
requests-oauthlib         1.3.0               
rsa                       4.0                 
s3transfer                0.5.0               
scipy                     1.4.1               
Send2Trash                1.5.0               
setuptools                44.0.0.post20200106 
six                       1.14.0              
soupsieve                 2.0                 
tabulate                  0.8.6               
tdw                       1.8.29.0            
tensorboard               2.1.0               
tensorboardX              2.0                 
tensorflow                2.1.0               
tensorflow-estimator      2.1.0               
termcolor                 1.1.0               
terminado                 0.8.3               
testpath                  0.4.4               
toml                      0.10.0              
torch                     1.5.0a0+462bfc7     
tornado                   6.0.3               
tqdm                      4.62.3              
traitlets                 4.3.3               
typed-ast                 1.4.1               
typing                    3.6.4               
urllib3                   1.25.7              
wcwidth                   0.1.7               
webencodings              0.5.1               
Werkzeug                  0.16.0              
wheel                     0.33.6              
widgetsnbextension        3.5.1               
wrapt                     1.12.1              
yarl                      1.4.2               
zipp                      3.1.0               
zmq                       0.0.0
